tensorflow-core.yaml

```diff
@@ -226,28 +236,5 @@ update:
     tag-filter: v
 
 test:
-  environment:
-    contents:
-      packages:
-        - py3-tensorflow-core
```

Comment:
This will crash on the new instance types and isn't built any longer (plus the commands below are handled in the subpackages now).